2-mandatory/1-weather-report.js

```diff
@@ -12,9 +12,17 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
-}
+if (cities === []) {   // TODO
+return [];
```

Comment:
You don't even need the check for empty array because it will return empty anyways if its passed as empty array

Reply:
Thank you @AltomHussain. yes you are right